src/main/java/org/commcare/formplayer/api/json/PromptToJson.java

```diff
@@ -44,6 +44,9 @@ public static void parseQuestion(FormEntryPrompt prompt, JSONObject questionJson
 
         if (prompt.getDataType() == Constants.DATATYPE_CHOICE || prompt.getDataType() == Constants.DATATYPE_CHOICE_LIST) {
             questionJson.put("choices", parseSelect(prompt));
+            // this creates an improved choices object that contains both values and captions for each choice
+            JSONArray choicesWithCaptions = parseChoicesWithCaptions(prompt);
+            questionJson.put("choices_v2", choicesWithCaptions);
```

Comment:
Is there a HQ PR that handles this attribute, If yes, would you be able to link it the PR description.

Reply:
Yeah it's currently on a branch locally. Figured I wouldn't PR it until this was set in stone so I'll make that today and link it.